src/Annotation/VirtualProperty.php

```diff
@@ -12,24 +12,25 @@
  *
  * @author Alexander Klimenkov <alx.devel@gmail.com>
  */
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
 final class VirtualProperty
 {
     /**
-     * @var string
+     * @var string}null
```

Comment:
I missed the typo here.

```suggestion
     * @var string|null
```